client/landing/site.landing/styl/team.styl

```diff
@@ -12,46 +13,67 @@ team.styl
 */
 }
 
+body.team #kdmaincontainer #main-tab-view
+body.teams #kdmaincontainer #main-tab-view
```

Comment:
this would cause heavy repetition on the compiled css, can you pls make `body` to share a common class for both cases and use it here.